src/com.c

```diff
@@ -272,7 +272,7 @@ static void print_version(bool sync)
 void COM_init(void)
 {
 	print_version(false);
-#if (defined COM_RS232) || (defined COM_RS485) && (THERMOTRONIC != 1)
+#ifdef COM_RS232

```

Comment:
This looks to be changing a bit more than removing RS485? Was the THERMOTRONIC define not needed here?

Reply:
I'm quite sure, that I did this change on purpose, but I'll recheck this!

Reply:
Yay, found the thing: [config.h:78](https://github.com/OpenHR20/OpenHR20/blob/689f7f81263de1de7e17995f6653273bd250f57e/rfmsrc/OpenHR20/config.h#L78)
As the config says, Thermotronic doesn't implement UART, so all preprocessor ifdefs for Thermotronic can be deleted from com.c in my opinion. What do you think?

Reply:
Yes, you are right.